src/components/Counter/tests/counter.test.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
```

Comment:
Please rename `tests` folder into `__tests__` for each component

Reply:
Done. Thanks.